packages/block-library/src/index.native.js

```diff
@@ -148,7 +148,7 @@ export const registerCoreBlocks = () => {
 		mediaText,
 		preformatted,
 		gallery,
-		devOnly( group ),
+		group,
```

Comment:
I know that we want to enable this on production, but this should go in its own PR

Reply:
it was added to this PR because of discussion on Slack but its not a problem to do this separately

Reply:
done